pkgs/applications/networking/mailreaders/alot/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, pkgs, pythonPackages
+, withManpage ? true }:
+
+with pythonPackages;
+
+buildPythonApplication rec {
+    rev = "0.6";
+    name = "alot-${rev}";
+    outputs = [ "out"  "man" ];
+
+    disabled = isPy3k;
+
+    src = pkgs.fetchFromGitHub {
+      owner = "pazz";
+      repo = "alot";
+      inherit rev;
+      sha256 = "1x6in9y823zi0k6xk62nwjybkc3pp5n2ml79x4v9jy6fz2klf2mi";
+    };
+
+    postPatch = ''
+      substituteInPlace alot/defaults/alot.rc.spec \
+        --replace "themes_dir = string(default=None)" \
+                  "themes_dir = string(default='$out/share/themes')"
+    '';
+
+    checkInputs =  [ mock pkgs.gnupg ];
+    nativeBuildInputs = stdenv.lib.optionals withManpage [ sphinx ];
+
+    propagatedBuildInputs = [
+        notmuch
+        urwid
+        urwidtrees
+        twisted
+        python_magic
+        configobj
+        gpg
+      ];
+
+    doCheck = false;
+    postBuild= stdenv.lib.optionalString withManpage "make -C docs man";
```

Comment:
rather then a build option, I would add `outputs = ["out" "man"]` to split the build and move the generated documentation there.